src/main/java/org/jabref/logic/util/io/DatabaseFileLookup.java

```diff
@@ -66,4 +69,8 @@ private List<Path> parseFileField(BibEntry entry) {
                     .map(Optional::get)
                     .collect(Collectors.toList());
     }
+
+    public Path getPathOfDatabase() {
+        return pathOfDatabase;
+    }
```

Comment:
The most clean code would be to use `Optional<Path>` here. However, the only place using this would be more complex. @Siedlerchr Any oppinion here?

If this is kept as is, the JavaDoc comment should be put here

```
@returns "" if the Path does not exist
```

Reply:
This is just wrong. A Path object can never be an empty string.

Reply:
We should return `null` then? Then, the calling code would be "clean". Think this method is never reached if the database is not saved.